src/comparator/service_comparator.py

```diff
@@ -124,6 +124,42 @@ def _compareRpcMethods(
             signatures_update = self._get_signatures(method_update)
             self._compare_method_signatures(signatures_original, signatures_update)
 
+            # 6.9 The LRO operation_info annotation is changed.
+            lro_original = self._get_lro(method_original)
+            lro_update = self._get_lro(method_update)
+            self._compare_lro_annotations(lro_original, lro_update)
+
+    def _get_lro(self, method: MethodDescriptorProto):
+        """Return the LRO operation_info annotation defined for this method."""
+        if not method.output_type.endswith("google.longrunning.Operation"):
+            return None
+        op = method.options.Extensions(operations_pb2.operation_info)
+        if not op.response_type or not op.metadata_type:
+            raise TypeError(
+                f"rpc {method.name} returns a google.longrunning."
+                "Operation, but is missing a response type or "
+                "metadata type.",
+            )
+        return [op.response_type, op.metadata_type]
+
+    def _compare_lro_annotations(self, lro_original, lro_update):
+        if not lro_original or not lro_update:
+            return
```

Comment:
Here we only detect the LRO annotation change (response_type or metadata_type change) because the annotation is associated with the output_type of the method and we assume the protos cannot be compiled correctly. 
1) LRO annotation removal: if the original is LRO method, but the updated method is not LRO, then the output_type of the updated should not be `google.longrunning.Operation` and this change will be caught by RPC method check #6.4 in this file.
2) LRO annotation addition: similar as 1) the output_type change will be caught be RPC method comparison.

Reply:
On a second thought, I think that makes sense to also remind users that the LRO annotation is removed/added with the output_type change. Updated the annotation detection, if the LRO annotation is either removed/added, that would also result in breaking changes. Thanks!